source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,355 @@
+| Incident Response | https://github.com/ministryofjustice/modernisation-platform-incident-response | | No |
+| Trusted Advisor | https://github.com/ministryofjustice/modernisation-platform-terraform-trusted-advisor | | No |
+| Lambda Function | https://github.com/ministryofjustice/modernisation-platform-terraform-lambda-function | | No |
+| | | | |
+
+
+
+### Organisational Service Control Policies
+
+
+Service control policies (SCPs) are a type of organization policy that you can use to manage permissions in the organization. SCPs offer central control over the maximum available permissions for all accounts in your organization. SCPs help you to ensure your accounts stay within your organization’s access control guidelines.
+
+
+Currently in `eu-west-1` we can build what is required, however if this region was to change, we would need to check what is limited in which region.
+
+
+### Contact AWS
+
+
+As previously mentioned, contact AWS using the following steps
+
+
+- Raise a ticket, highest priority.
+- Contact our AWS representative for an update on the situation.
+- Organise updates, if the situation changes, our approach might need to.
+
+
+### Running the terraform
+
+
+Once the modules have been updated, and tagged, as well as the PR merged to change any variable or hardcoded mentions of region in `Modernisation-Platform` we will need to begin the rebuild of the platform, starting with networking. The steps below outline which order to follow.
+
+
+## Platform rebuild - Networking
+
+
+This is a large undertaking, and has not been tested but discussed within the team.
+
+
+### Networking Overview
+
+
+Our [Networking Diagram](../concepts/networking/networking-diagram.html) shows a high level view of how shared
+VPCs are connected.
+
+
+Our [NetworkingApproach](../concepts/networking/networking-approach.html) discusses and
+contains a detailed view of our shared VPCs.
+
+
+Our [Transit Gateway](../how-vpcs-access-the-internet.html) and how VPCs access the internet.
+
+
+
+## Stages
+
+Rather than rebuilding the platform through workflows, I would run steps manually to easier debug issues, however, technically, you could start to run the account creation process with our github workflows.
+
+Located [here](https://github.com/ministryofjustice/modernisation-platform/actions), on the left you will see the deployment workflows for all core based accounts, required for the building on the platform.
+
+
+### VPCs and Subnets
+
+
+1. Clone the mod-platform repo, change region in code. Whilst its as simple as a find and replace, modules are called and need changing, either locally and referenced locally, or in the module and pushed to git.
+2. Core VPC rebuild. Provider would need to be eu-west-1, so check the permissions you have in that region with the credentials you are using. E.g. https://moj.awsapps.com/start/ when getting your credentials from here, make sure the region is correct.
```

Comment:
I wonder if we should not specify the region we migrate to, as we do not know for sure it will be `eu-west-1`.
We could mention somewhere at the top that our preference will be `eu-west-1`, but it may not be possible to migrate to there at the time, hence for the sake of documentation we will call the new region `secondary region` or `other region` (or similar).